### ISSUE TITLE
Can send messages without errors but others do not see my messages

### ISSUE THREAD

**Issue by SylvainCecchetto** [matrix-org/synapse#4018]
### Description

My responses and status modification are not visible by other peoples of the rooms, but my Riot says that my message has correctly be send without any error.
If I leave a room I still be present on the room for other peoples.
If I modify my role level it has no effect for other peoples on the room.
If a create a new user he has the same issues.
But I can see all other people's messages.
And other peoples can see that I am writing and also that I read the message (the little badge) but not my sent message.

My matrix worked before. this problem

Thank you!

### Steps to reproduce

- Just log in with Riot on my account

### Version information

- **Homeserver**: My instance lives on matrix.cecchettosylvain.fr

- **Version**: 0.33.5.1

- **Install method**: YunoHost (https://github.com/YunoHost-Apps/synapse_ynh)      
- **Platform**: YunoHost server (based on Debian) at home

**Comment by richvdh** [comment]
please can you share logs.

**Comment by SylvainCecchetto** [comment]
Hi!

Yes, this my log in DEBUG level (User sy6sy2 from matrix.org can't see messages from sylvain user on matrix.cecchettosylvain.fr)

[homeserver.log](https://github.com/matrix-org/synapse/files/2460281/homeserver.log)

**Comment by richvdh** [comment]
thanks.

can you try the query at https://github.com/matrix-org/synapse/issues/3750#issuecomment-426718203

**Comment by SylvainCecchetto** [comment]
Yes

```
psql (9.6.10)
Saisissez « help » pour l'aide.

matrix_synapse=> SELECT COUNT(*) FROM federation_stream_position;
 count
-------
     6
(1 ligne)

matrix_synapse=>
```

**Comment by richvdh** [comment]
thanks. I'd like to discuss this more with you. can you message me at `@richvdh:sw1v.org` (from an account with working federation!)

**Comment by richvdh** [comment]
Resolved via pm; I have opened #4023 to track the underlying bug here.
